pubsub/natsstreaming/nats-streaming.go

```diff
@@ -0,0 +1,239 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package natsstreaming
+
+import (
+	"errors"
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+
+	"github.com/dapr/components-contrib/pubsub"
+	nats "github.com/nats-io/nats.go"
+	stan "github.com/nats-io/stan.go"
+	"github.com/nats-io/stan.go/pb"
+)
+
+const (
+	//compulsory
```

Comment:
nit: Would you follow [this](https://golang.org/doc/effective_go.html#commentary) commenting style if possible? Also I would add one space after `//`.

Reply:
done